### DISCUSSION TITLE
Stuck in boot screen after update

### DISCUSSION THREAD

**Discussion by MacUser341** [daliansky/XiaoMi-Pro#546] [Q&A]
Hello
I have a dual boot with windows 10 and Mac OS X on single drive with Open Core.
After an update to Mac OS when I’m trying to boot into Mac I get the boot screen stuck , I can still boot to windows .
How can I fix it ?

<img width="694" alt="82F8FAFE-734D-481D-B7AC-B3F5C0DBD837" src="https://user-images.githubusercontent.com/79283835/108555960-ee49b680-72fe-11eb-8b24-334a35010b09.png">

**Comment by stevezz3** [comment]
@MacUser341 Hi, please give more details about your model, macOS version and EFI version.
You can first try to reset NVRAM by pressing `Space` at OC boot page and choosing Reset NVRAM.
If the problem still exists, I suggest adding `-v` in `config - NVRAM - Add - 7CXXX - boot-args` and take a screenshot when the device got stuck.
Also make sure you are using v1.5.2 EFI release(BOOT and OC are completely replaced, which means delete and copy) and there's no PM981 disk.
